### ISSUE TITLE
Sometime coloring of random characters

### ISSUE THREAD

**Issue by biobshub** [madskristensen/RainbowBraces#28]
When editing a code file  (.cs, or javascript) , after some changes the extension is not only coloring brackets, but 
also random characters in any words.
after saving the file the coloring is normal again.

it is hard to describe how to reproduce, because it happens randomly.
very often after automated refactoring or cut/paste  code blocks.

system is VS 2022 17.3.4  with latest Resharper tools.

**Comment by madskristensen** [comment]
I cannot reproduce this with the latest version 1.0.37, but I did see it in an earlier version. Can you verify that the issue still reproduces for you on the version 1.0.37 or if it can be closed as fixed?

**Comment by biobshub** [comment]
yes, it happens on version 1.0.37.  because the occurrence is so randomly, i still cannot give a hint how to reproduce it.
last time it happened after automatic refactoring/renaming a c# class. 

btw. thanks for the tool, it is very useful.

**Comment by bmadtiger** [comment]
Just happened to me on a fresh install after editing the parameter list of a method:
![image](https://user-images.githubusercontent.com/49149264/192767997-b41f1673-2f72-4542-80a3-66fa1bb03623.png)
Also note the nesting levels of the brackets are out of sync (see also #31).

NB Scrolling refreshes the screen and resolves the issue.

**Comment by gnimor** [comment]
I can see the same behaviour here - also after a fresh install without any other extension installed.

**Comment by twoelfer** [comment]
I see the same random-colored letters problem with 1.0.37.. What i also see: When some letters are displayed in a random color and i simply scroll down and up again (the letters do not need to go offscreen), the correct color is used.

**Comment by get-flat** [comment]
Another scenario: correcting function definition with quick actions (no parenthesis editing, adding/removing)

Before:  ```private void SomeFunction(MyType type)```
After: Use VS Quick Actions to fully qualify ```MyType``` name i.e. ```MyClass.MyType``` (after quick actions applied: ```private void SomeFunction(MyClass.MyType type)``` results in random characters coloring (not only in this line). Scrolling through code helps

Version 1.0.40

**Comment by sholman** [comment]
I'm seeing this on version 1.0.40 as well.
